src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -685,6 +675,47 @@ private void getCanvasResponse(HttpServletRequest request,
 		out.flush();
 	}
 
+	//When there is too much data to retrieve at once data will be paged.
+	//When data is paged Link Headers will be send in http response. We are
+	//only concerned with the Next Link Header. This method will extract header 
+	//if one exists.
+	private String extractNextLink(HttpResponse canvasResponse) {		
+		String linkValueString = null;
+		String linkValueNext = null;
+		String searchPhrase = "rel=\"next\"";
+		M_log.debug("SEARCH PHRASE: " + searchPhrase);
+		HashMap<String, String> links = new HashMap<String, String>();
+		Header[] headers = canvasResponse.getHeaders("Link");
+		for (Header header : headers) {
+			M_log.debug("Key : " + header.getName() 
+					+ " ,Value : " + header.getValue());
+			if(header.getName().equals("Link")){
```

Comment:
Since you're using `canvasResponse.getHeaders("Link")` now, you don't need this if statement.  You can simplify the logic.  ;)